docs/notebooks/expected_improvement.pct.py

```diff
@@ -66,6 +66,9 @@ def build_model(data):
 
     return {
             "model": gpr,
+            "model_args": {
```

Comment:
Do we want to mention setting num_prior_samples in the text above (where we talk about packaging it with the gpflow scypy optimizer?